flink-core/src/main/java/org/apache/flink/configuration/RestOptions.java

```diff
@@ -32,13 +32,11 @@
 	 * The address that the server binds itself to / the client connects to.
 	 */
 	public static final ConfigOption<String> REST_ADDRESS =
-		key("rest.address")
-			.defaultValue("localhost");
+		key("rest.address").defaultValue("localhost");
 
 	/**
 	 * The port that the server listens on / the client connects to.
 	 */
 	public static final ConfigOption<Integer> REST_PORT =
-		key("rest.port")
-			.defaultValue(9067);
+		key("rest.port").defaultValue(9065);
```

Comment:
If the change of the default port is intentional, we should also change the documentation:
https://ci.apache.org/projects/flink/flink-docs-master/dev/stream/state/queryable_state.html

Reply:
This does not change the QS server port but the Rest server endpoint port.

Reply:
You are right!